main.py

```diff
@@ -55,15 +54,24 @@
 emptyID = config["statics"]["emojisIDs"]["empty"]
 debugMode = config["debugMode"]
```

Comment:
you can make constants to look like constants, it's easier to track and handle them, specially if they are stored in separate file:
```suggestion
DEBUG_MODE = config["debugMode"]
```